gef.py

```diff
@@ -8848,6 +8848,60 @@ def do_invoke(self, argv: List[str]) -> None:
             err("No address mapping information found")
             return
 
+        # Parse args:
+        #  - `-a` / `--addr`:
+        #    - filter by address -> parses the next arg as an int or asks gdb
+        #    the value
+        #  - `-n` / `--name`:
+        #    - filter based on section name
+        #  - If nothing is specified, print a warning and guess the type
```

Comment:
This is not necessary to have in code (sufficiently explicit), it can be removed

Reply:
Actually this would make more sense to be in the docs rather than in the code